contracts/PrivateContract.sol

```diff
@@ -72,4 +79,21 @@ contract PrivateContract {
 		state = newState;
 		nonce = nonce + 1;
 	}
+	
+	function notifyChanges(
+		address changesOriginator,
+		bytes originalTransactionHash
+	)
+		public
```

Comment:
I don't know much about this contract but should this be callable by anyone?

Reply:
@andresilva you're the reviewer of the other PR, in which call for this method added :-)
https://github.com/paritytech/parity-ethereum/pull/9761/
In particular: https://github.com/paritytech/parity-ethereum/pull/9761/files#diff-51c58eac52911d8404ca94c56fe7818dR366

Reply:
Yep, that's how I got here in the first place :wink: My point still stands though, shouldn't we somehow restrict who can call this method? It seems that in our use case the transaction will be sent from null sender.

Reply:
Aaa, got it. As for your question:
1) It's not so easy to restrict, because it may be called by any participant, that has has access to the keys for this private contract. Till now we haven't had any mapping to store list of participants inside this contract. This would complicate the system a lot, now this participants set is managed via Secret Store permissioning contract, so we would need to sync these lists somehow. 
2) it's only the convenient notification for observers, frankly I don't see the serious reasons to secure it.